djangoproject/templates/fundraising/index.html

```diff
@@ -118,12 +119,13 @@ <h2 id="fellowship-program">{% translate "Django Fellowship Program" %}</h2>
   </p>
 
   <p>
+    {% host_url 'homepage' host 'www' as base_url %}
     {% blocktranslate trimmed %}
       For more details, you can read retrospectives for the
-      <a href="/weblog/2015/jan/21/django-fellowship-retrospective/"> first three
+      <a href="{{ base_url }}weblog/2015/jan/21/django-fellowship-retrospective/"> first three
```

Comment:
What about using the actual URL name for this? Do you find it too verbose?
```suggestion
      <a href="{% host_url 'weblog:entry' year=2015 month='jan' slug='django-fellowship-retrospective' host 'www' %}">first three
```